qt/scientific_interfaces/General/SampleTransmission.cpp

```diff
@@ -82,6 +82,20 @@ bool SampleTransmission::validate(bool silent) {
   case 1:
     uiv.checkFieldIsNotEmpty("Multiple binning", m_uiForm.leMultiple,
                              m_uiForm.valMultiple);
+
+    int firstChararacterAfterWhitespaceIndex{static_cast<int>(
+        m_uiForm.leMultiple->text().toStdString().find_first_not_of(' '))};
+
+    // checks that the first number in the list is not negative
+    // only need to check the first number because if it isn't negative then the
+    // rest can't be
+    // the Rebin algorithm takes care of the rest of the checks
```

Comment:
A bit picky, but could we clean up this comment with capital letters and punctuation?

Reply:
Fixed the comment and also removed the cast because I realised it isn't needed

Reply:
Martyn told me that it's best to avoid using toStdString() so I've now taken a different approach